charts/sonarqube/Chart.yaml

```diff
@@ -26,33 +26,11 @@ annotations:
       url: https://github.com/SonarSource/helm-chart-sonarqube/tree/master/charts/sonarqube
   artifacthub.io/changes: |
     - kind: changed
-      description: "Updated SonarQube to 9.7.0"
```

Comment:
you can revert this change as we are not releasing from this branch

Reply:
All reverted!